sdk/openai/openai/src/models/models.ts

```diff
@@ -1727,6 +1728,34 @@ export interface EmbeddingsUsage {
   /** Total number of tokens transacted in this request/response. */
   totalTokens: number;
 }
+/**
+ * The options supported by OpenAIError.
+ */
+export interface OpenAIErrorOptions extends RestErrorOptions {
+  /** The parameters of the error. */
+  param: string | null;
+  /** Type of the error. */
+  type: string | null;
+}
```

Comment:
It feels weird that these values are part of an options bag but are not optional. Did you consider adding them to the parameter list of the class constructor?

Reply:
That's a good catch. This model was designed when `param` and `type` are optional string type, so it'd make more sense for them to be in the class constructor.